deployments/examples/ocis_wopi/docker-compose.yml

```diff
@@ -88,8 +88,10 @@ services:
       MICRO_REGISTRY_ADDRESS: 127.0.0.1:9233
       NATS_NATS_HOST: 0.0.0.0
       NATS_NATS_PORT: 9233
+      PROXY_CSP_CONFIG_FILE_LOCATION: /etc/ocis/csp.yaml
```

Comment:
@DeepDiver1975 
Shouldn't `ONLYOFFICE_DOMAIN` and `COLLABORA_DOMAIN` be set here as well in order for the substitution in `etc/ocis/csp.yaml` to take effect?

```yaml
ONLYOFFICE_DOMAIN: ${ONLYOFFICE_DOMAIN:-onlyoffice.owncloud.test}
COLLABORA_DOMAIN: ${COLLABORA_DOMAIN:-collabora.owncloud.test}
```

Reply:
see https://github.com/owncloud/ocis/pull/8777/files#diff-bfedf12f7818c318b754f308bff1d6139afd701f2fc7bb43f07fd865a0d85039R15

Reply:
@DeepDiver1975 Yes, I mean the `csp.yaml` is being loaded inside the containers environment, so if the `ONLYOFFICE_DOMAIN`  and `COLLABORA_DOMAIN` are not set in the `docker-compose.yml` file they will be empty during runtime and the default values are used. (This is at least what happened when I tried it.)

Reply:
yes - but the env vars are not forwarded into the ocis container and will always be set to the default.
pr coming ...

Reply:
Yes, right

Reply:
THX a lot @kjeldahl for bringing this to our attention! :pray:

Reply:
@DeepDiver1975 You are most welcome. And thanks a lot for solving the CSP issue 4 hours before I ran into it 👍